### PR TITLE
[develop] Update the hashes of the UFS-WM and UFS_UTILS repositories

### LINKED ISSUE #855: CCPP suites that have been dropped from the ufs-weather-model and the WE2E tests that are associated with them
## Description
As of hash [18c44f1](https://github.com/ufs-community/ufs-weather-model/commit/18c44f1641720f65d27361610fc0eb673cc0e0b9) in the ufs-weather-model, the SRW App will fail to build because CCPP suites that aren't tested in the ufs-weather-model's regression tests have been moved to a suites_not_used directory.  The SRW App is currently using both the `FV3_GFS_2017_gfdlmp` and `FV3_GFS_2017_gfdlmp_regional` suites, which have been moved.  Removing these two suites allows the SRW App to build without issue.

## Solution
Remove the `FV3_GFS_2017_gfdlmp` and `FV3_GFS_2017_gfdlmp_regional` suites from the sorc/CMakeLists.txt and remove tests that use these CCPP suites.

## Acceptance Criteria (Definition of Done)
The SRW App builds and all tests run using the current HEAD of the ufs-weather-model repository.

### PR DESCRIPTION
**Pull request by MichaelLueken**
## DESCRIPTION OF CHANGES: 
Update the hash of the UFS-WM to [24c51d1](https://github.com/ufs-community/ufs-weather-model/commit/24c51d196ed7ec97c4256f7e78a5b0466b52937e) from July 5 and UFS_UTILS to [ae483bd](https://github.com/ufs-community/UFS_UTILS/commit/ae483bd31d8fd81d35c3c726e746c55c46a3ef61) from June 26.  Additionally, remove Cheyenne from the Jenkinsfile since EPIC no longer has usable allocation on that machine (all core-hours have been spent).

### Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## TESTS CONDUCTED: 
- [X] hera.intel
- [X] hera.gnu
- [X] orion.intel
- [X] cheyenne.intel
- [X] cheyenne.gnu
- Cheyenne tests coverage WE2E tests were manually ran on Hera and all passed.
- [ ] gaea.intel
- [X] jet.intel
- [ ] wcoss2.intel
- [ ] NOAA Cloud (indicate which platform)
- [ ] Jenkins
- [X] fundamental test suite
- [X] coverage test suite
- [ ] comprehensive tests (specify *which* if a subset was used)

## ISSUE: 
Fixes #855

## CHECKLIST
- [X] My code follows the style guidelines in the Contributor's Guide
- [X] I have performed a self-review of my own code using the Code Reviewer's Guide
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [X] My changes do not require updates to the documentation (explain).
- [X] My changes generate no new warnings
- [X] New and existing tests pass with my changes
- [ ] Any dependent changes have been merged and published

### CODE REVIEW

**Comment by MichaelLueken** [comment]
The Cheyenne Intel WE2E coverage tests were manually ran on Hera and all tests passed:

```
----------------------------------------------------------------------------------------------------
Experiment name                                                  | Status    | Core hours used 
----------------------------------------------------------------------------------------------------
custom_GFDLgrid__GFDLgrid_USE_NUM_CELLS_IN_FILENAMES_eq_FALSE      COMPLETE               7.54
grid_RRFS_CONUS_13km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16_plot     COMPLETE              29.39
grid_RRFS_CONUS_25km_ics_NAM_lbcs_NAM_suite_GFS_v16                COMPLETE              20.60
grid_RRFS_CONUScompact_13km_ics_HRRR_lbcs_RAP_suite_HRRR           COMPLETE              24.90
grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta    COMPLETE               8.57
grid_SUBCONUS_Ind_3km_ics_HRRR_lbcs_HRRR_suite_HRRR                COMPLETE              16.18
pregen_grid_orog_sfc_climo                                         COMPLETE               6.83
specify_template_filenames                                         COMPLETE               6.62
----------------------------------------------------------------------------------------------------
Total                                                              COMPLETE             120.63
```
The Cheyenne GNU WE2E coverage tests were manually ran on Hera and all tests passed:

```
----------------------------------------------------------------------------------------------------
Experiment name                                                  | Status    | Core hours used 
----------------------------------------------------------------------------------------------------
grid_CONUS_25km_GFDLgrid_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16      COMPLETE              20.13
grid_RRFS_CONUS_13km_ics_FV3GFS_lbcs_FV3GFS_suite_RRFS_v1beta      COMPLETE             232.84
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v17_p8_plot  COMPLETE              27.06
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_HRRR             COMPLETE              36.22
grid_RRFS_CONUScompact_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16   COMPLETE              23.00
grid_RRFS_NA_13km_ics_FV3GFS_lbcs_FV3GFS_suite_RAP                 COMPLETE             319.95
grid_SUBCONUS_Ind_3km_ics_NAM_lbcs_NAM_suite_GFS_v16               COMPLETE              51.55
specify_EXTRN_MDL_SYSBASEDIR_ICS_LBCS                              COMPLETE              10.10
----------------------------------------------------------------------------------------------------
Total                                                              COMPLETE             720.85
```

File: tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_2017_gfdlmp.yaml [hunk 1]
```diff
@@ -1,27 +0,0 @@
-metadata:
-  description: |-
-    This test is to ensure that the workflow running in community mode
-    completes successfully on the RRFS_CONUS_25km grid using the GFS_2017_gfdlmp
-    physics suite with ICs and LBCs derived from the FV3GFS.
-    This test additionally includes the cycling ensemble capability, with
```
Comment (mkavulich) [inline tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_2017_gfdlmp.yaml:6]: The cycling ensemble capability needs to be added to another test with the loss of this one. I am not sure what the best candidate might be; there's also the option to just remove it temporarily and add it back with a future PR (my upcoming testing PR adds a few new tests)

Also, The test `wflow_features/config.community_ensemble_2mems.yaml` is a symlink to this one; you should replace that symlink to point to the test above.

Comment (MichaelLueken) [reply tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_2017_gfdlmp.yaml:6]: Thank you very much for bringing this to my attention, @mkavulich!  I could just bring this test back, change the `CCPP_PHYS_SUITE` to `FV3_GFS_v15p2`, rename the test to replace `GFS_2017_gfdlmp` with `GFS_v15p2`, then update the symlink for `wflow_features/config.comunity_ensemble_2mems.yaml` to point to the updated test.  We could then circle back later and replace the test with a new one entirely.

Comment (MichaelLueken) [reply tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_2017_gfdlmp.yaml:6]: I have added the test back, but am now having it use `FV3_RAP` CCPP physics suite.  I have also changed the symlink for `wflow_features/config.comunity_ensemble_2mems.yaml` to point to the new test.  I also noted that `wflow_feature/config.specify_RESTART_INTERVAL.yaml` was pointing at a test configuration that no longer exists, so I updated the symlink for this configuration too.

**Review by mkavulich — APPROVED** [review]
Thanks for addressing my comments, looks good to me.

Just a side question: will the Cheyenne coverage tests be run manually on another platform for the near future until we can get hours back for Cheyenne testing?

**Comment by MichaelLueken** [comment]
@mkavulich I will be manually running the Cheyenne coverage tests on Hera until EPIC can once again submit jobs on the machine.

**Review by natalie-perlin — APPROVED** [review]

**Review by DavidHuber-NOAA — APPROVED** [review]
Built and reran the fundamental tests on Hera.  Everything looks good.  Thanks @MichaelLueken!

**Comment by MichaelLueken** [comment]
Thanks for the reviews, @natalie-perlin and @DavidHuber-NOAA!

**Review by natalie-perlin — APPROVED** [review]

**Comment by MichaelLueken** [comment]
The Cheyenne Intel tests were manually run on Hera and all tests passed:
```
----------------------------------------------------------------------------------------------------
Experiment name                                                  | Status    | Core hours used 
----------------------------------------------------------------------------------------------------
custom_GFDLgrid__GFDLgrid_USE_NUM_CELLS_IN_FILENAMES_eq_FALSE      COMPLETE               7.26
grid_RRFS_CONUS_13km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16_plot     COMPLETE              29.00
grid_RRFS_CONUS_25km_ics_NAM_lbcs_NAM_suite_GFS_v16                COMPLETE              19.11
grid_RRFS_CONUScompact_13km_ics_HRRR_lbcs_RAP_suite_HRRR           COMPLETE              25.54
grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta    COMPLETE               8.36
grid_SUBCONUS_Ind_3km_ics_HRRR_lbcs_HRRR_suite_HRRR                COMPLETE              16.18
pregen_grid_orog_sfc_climo                                         COMPLETE               7.04
specify_template_filenames                                         COMPLETE               6.95
----------------------------------------------------------------------------------------------------
Total                                                              COMPLETE             119.44
```

**Comment by MichaelLueken** [comment]
The Cheyenne GNU tests were manually run on Hera and all successfully passed:
```
----------------------------------------------------------------------------------------------------
Experiment name                                                  | Status    | Core hours used 
----------------------------------------------------------------------------------------------------
grid_CONUS_25km_GFDLgrid_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16      COMPLETE              20.43
grid_RRFS_CONUS_13km_ics_FV3GFS_lbcs_FV3GFS_suite_RRFS_v1beta      COMPLETE             230.37
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v17_p8_plot  COMPLETE              27.17
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_HRRR             COMPLETE              36.95
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_RAP              COMPLETE              81.45
grid_RRFS_CONUScompact_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16   COMPLETE              22.47
grid_RRFS_NA_13km_ics_FV3GFS_lbcs_FV3GFS_suite_RAP                 COMPLETE             316.26
grid_SUBCONUS_Ind_3km_ics_NAM_lbcs_NAM_suite_GFS_v16               COMPLETE              50.01
specify_EXTRN_MDL_SYSBASEDIR_ICS_LBCS                              COMPLETE              10.08
----------------------------------------------------------------------------------------------------
Total                                                              COMPLETE             795.19
```
The Gaea WE2E coverage tests were manually run and all successfully passed:
```
----------------------------------------------------------------------------------------------------
Experiment name                                                  | Status    | Core hours used 
----------------------------------------------------------------------------------------------------
community                                                          COMPLETE              24.94
grid_RRFS_CONUScompact_13km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta    COMPLETE              30.29
grid_RRFS_CONUS_13km_ics_FV3GFS_lbcs_FV3GFS_suite_RAP              COMPLETE              30.16
grid_RRFS_CONUS_13km_ics_FV3GFS_lbcs_FV3GFS_suite_HRRR             COMPLETE              34.15
grid_RRFS_CONUS_3km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15_thompson  COMPLETE             373.31
grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_HRRR_suite_HRRR          COMPLETE              33.61
grid_RRFS_CONUScompact_3km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta     COMPLETE             351.42
grid_SUBCONUS_Ind_3km_ics_RAP_lbcs_RAP_suite_RRFS_v1beta_plot      COMPLETE              10.61
nco_ensemble                                                       COMPLETE              81.91
nco_grid_RRFS_CONUS_3km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15_thom  COMPLETE             353.14
----------------------------------------------------------------------------------------------------
Total                                                              COMPLETE            1323.54
```
The Jet WE2E coverage tests were manually run and all successfully passed:
```
----------------------------------------------------------------------------------------------------
Experiment name                                                  | Status    | Core hours used 
----------------------------------------------------------------------------------------------------
community                                                          COMPLETE              17.69
custom_ESGgrid                                                     COMPLETE              16.81
custom_GFDLgrid                                                    COMPLETE               9.20
get_from_HPSS_ics_FV3GFS_lbcs_FV3GFS_fmt_nemsio_2021032018         COMPLETE               9.62
get_from_HPSS_ics_FV3GFS_lbcs_FV3GFS_fmt_netcdf_2022060112_48h     COMPLETE              75.81
get_from_HPSS_ics_RAP_lbcs_RAP                                     COMPLETE              18.34
grid_RRFS_AK_3km_ics_FV3GFS_lbcs_FV3GFS_suite_HRRR                 COMPLETE             238.07
grid_RRFS_CONUS_13km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16_plot     COMPLETE              38.26
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2        COMPLETE               7.95
grid_RRFS_CONUS_3km_ics_FV3GFS_lbcs_FV3GFS_suite_RRFS_v1beta       COMPLETE             554.01
nco_grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_HRRR       COMPLETE              10.91
process_obs                                                        COMPLETE               0.26
----------------------------------------------------------------------------------------------------
Total                                                              COMPLETE             996.93
```
The Hera and Orion Jenkins tests successfully passed as well.  Moving forward with merging this work now.

**Pull request merged**